packages/salesforcedx-vscode-soql/package.json

```diff
@@ -205,7 +217,7 @@
       ],
       "editor/title": [
         {
-          "when": "resourceLangId == soql && resourceScheme == file",
+          "when": "resourceExtname == .soql",
```

Comment:
This means the button will show up for any *.soql file even if the user has assigned a different language to the file.